File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Prompt the pyramid height until its in range
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    // Create the pyramid
    for (int i = 1; i <= height; i++) // Iterate the rows
    {
        for (int j = 1; j <= height; j++) // Iterate the columns
        {
            if (i + j <= height) // Left side of the pyramid
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        printf("  ");
        for (int k = 1; k <= i; k++) // Right side of the pyramid
        {
            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

for i in range(height):
    print(" " * int(height-(i+1)), end="")
    print("#" * int(i+1), end="  ")
    print("#" * int(i+1))
